run_data.py

```diff
@@ -25,27 +22,30 @@ def main():
 
 @main.command("latest")
 @click.option(
-    "--deploy",
-    is_flag=True,
-    help="Output data files to public data directory in local covid-projections.",
+    "--output",
+    "-o",
+    help="Output directory",
+    type=pathlib.Path,
+    default=pathlib.Path("results/case_summaries"),
 )
-def run_latest(deploy=False):
+@data_version.with_git_version_click_option
+def run_latest(version: data_version.DataVersion, output: pathlib.Path):
     """Get latest case values from JHU dataset."""
-    output_dir = pathlib.Path(build_params.OUTPUT_DIR)
-    if deploy:
-        output_dir = WEB_DEPLOY_PATH
-
-    output_folder = output_dir / "case_summary"
-    output_folder.mkdir(exist_ok=True)
+    output.mkdir(exist_ok=True)
     timeseries = JHUDataset.local().timeseries()
     state_summaries = dataset_export.latest_case_summaries_by_state(timeseries)
 
     for state, state_summary in state_summaries:
-        output_file = output_folder / f"{state}.summary.json"
+        output_file = output / f"{state}.summary.json"
         with output_file.open("w") as f:
             _logger.info(f"Writing latest data for {state}")
             json.dump(state_summary, f)
 
+    if not state:
```

Comment:
I don't think we want this condition here. `state` will always be defined, and I think we always want to write the version file since there doesn't appear to be an option to do a partial run.

Reply:
oh good call